### PR TITLE
JaydenDev.Catalyst3 version 3.2.2

### PR DESCRIPTION
**Pull request by jdev082**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86063)

### CODE REVIEW

**Comment by JaydenDev** [comment]
@microsoft-github-policy-service agree [company="JaydenDev"]

**Comment by JaydenDev** [comment]
@microsoft-github-policy-service agree

File: manifests/j/JaydenDev/Catalyst3/3.2.2/JaydenDev.Catalyst3.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JaydenDev.Catalyst3
+PackageVersion: 3.2.2
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://github.com/JaydenDev/Catalyst/releases/download/v3.2.2/catalyst-3.2.2.Setup.exe
+  InstallerSha256: 9D950FB4A4AD449FD8DF4E8F12FC1B1D31E4D9FCAF93D3BB2BB1767B86636F26
```
Comment (ItzLevvie) [inline manifests/j/JaydenDev/Catalyst3/3.2.2/JaydenDev.Catalyst3.installer.yaml:10]:
```suggestion
  InstallerSha256: 9D950FB4A4AD449FD8DF4E8F12FC1B1D31E4D9FCAF93D3BB2BB1767B86636F26
  AppsAndFeaturesEntries:
  - DisplayName: catalyst
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by JaydenDev** [comment]
@ItzLevvie the changes that you have suggested have been applied.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**